config/clusters/cloudbank/avc.values.yaml

```diff
@@ -1,4 +1,38 @@
 jupyterhub:
+    load_groups:
```

Comment:
This is JupyterHub Helm chart configuration, and it will error if you put `load_groups` here under `jupyterhub`.

You can see the [chart configuration reference](https://z2jh.jupyter.org/en/stable/resources/reference.html) for details, there is `hub.loadRoles` but not `hub.loadGroups`.

You can configure jupyterhub configuration directly via `hub.config.JupyterHub` though!

Reply:
tracking this down! Thanks. You were on that one! I am wondering if there is any documentation on testing the hub configuration files locally?  It is embarrassing to commit all over the place trying to figure out how it all comes together.